.jenkins

```diff
@@ -0,0 +1,166 @@
+pipeline {
+
+  agent any
+
+  environment {
+    IMAGE_NAME = 'vaporio/synse-server'
+  }
+
+  stages {
+    stage('Checkout') {
+      steps {
+        checkout scm
+      }
+    }
+    // Lint the project source code.
+    stage('Lint') {
+      steps {
+        sh 'make lint'
+      }
+      post {
+        success {
+          script {
+            if (env.CHANGE_ID) {
+              pullRequest.createStatus(
+                status: 'success',
+                context: 'vio-build/lint',
+                description: 'Source code linting succeeded',
+                targetUrl: "${env.JOB_URL}"
+              )
+            }
+          }
+        }
+        failure {
+          script {
+            if (env.CHANGE_ID) {
+              pullRequest.createStatus(
+                status: 'failure',
+                context: 'vio-build/lint',
+                description: 'Source code linting failed',
+                targetUrl: "${env.JOB_URL}"
```

Comment:
Instead of `${env.JOB_URL}` use `${env.RUN_DISPLAY_URL}` which will take you to the Blue Ocean page and not the broke ass Jenkins legacy page